api/src/certification/configuration/domain/usecases/create-consolidated-framework.js

```diff
@@ -32,5 +32,13 @@ export const createConsolidatedFramework = async ({
 
   const challenges = await challengeRepository.findOperativeBySkills(skills, LOCALE.FRENCH_SPOKEN);
 
-  return consolidatedFrameworkRepository.create({ complementaryCertificationKey, challenges });
+  return consolidatedFrameworkRepository.create({
+    complementaryCertificationKey,
+    challenges,
+    version: getVersionNumber(),
+  });
 };
+
+function getVersionNumber() {
+  return new Date().toISOString().slice(0, 19).replace(/-|T|:/g, '');
```

Comment:
Je trouve cette ligne un peu dure a lire, je propose une alternative que je lis + facilement (possible aussi de passer un date.format, mais on utilise globalement dayjs sur l'API).


```suggestion
  return dayjs().format('YYYYMMDDHHmmss');
```

Reply:
J'aimerais bien ne pas utiliser de bibliothèque externe pour un truc simple. Mais on peut regarder pour la rendre plus lisible...

Reply:
La bibliothèque est déjà présente dans l'API. On ne vient pas rajouter de dépendances, je ne vois pas le problème 🤔

Reply:
Ben si, le fait de l'utiliser ajoute une dépendance supplémentaire.

Nous avons changé l'implémentation pour rendre le mécanisme plus explicite. On pousse comme ça ! :)